tests/test_transforms/test_math/test_statistics_transform.py

```diff
@@ -177,6 +227,7 @@ def test_std_feature(simple_df_for_agg: pd.DataFrame, window: int, periods: int,
         MeanTransform(in_column="target", window=5),
         StdTransform(in_column="target", window=5),
         StdTransform(in_column="target", window=5),
```

Comment:
May you also remove duplicates from the list of parameters here:pray:

Reply:
done